sycamore/data/document.py

```diff
@@ -197,19 +173,19 @@ def binary_representation(self) -> None:
     def elements(self) -> list[Element]:
         """A list of elements belonging to this document. A document does not necessarily always have
         elements, for instance, before a document is chunked."""
-        return [Element(element) for element in self.data["elements"]["array"]]
+        return [Element(element) for element in self.data.get("elements", [])]
 
     @elements.setter
     def elements(self, elements: list[Element]):
-        self.data["elements"] = {"array": [element.to_dict() for element in elements]}
+        self.data["elements"] = [element.data for element in elements]
 
     @elements.deleter
     def elements(self) -> None:
-        self.data["elements"] = {"array": []}
+        self.data["elements"] = []
 
     @property
     def embedding(self) -> list[list[float]]:
-        return self.data["embedding"]
+        return self.data["embedding"] if "embedding" in self.data else None
```

Comment:
Should we return None for other properties as well when they do not exist?

Reply:
get would by default return none if it's missing.